src/main/java/com/iexec/worker/result/ResultService.java

```diff
@@ -339,9 +205,13 @@ public boolean isResultAvailable(String chainTaskId) {
         boolean isResultZipFound = isResultZipFound(chainTaskId);
         boolean isResultFolderFound = isResultFolderFound(chainTaskId);
 
-        if (!isResultZipFound && !isResultFolderFound) return false;
+        if (!isResultZipFound && !isResultFolderFound) {
+            return false;
+        }
 
-        if (!isResultZipFound) zipResultFolder(chainTaskId);
+        if (!isResultZipFound) {
```

Comment:
I could be wrong but ziping IexecOut again here looks like a patch (Sorry if it is related to E2E wrokflow). It probably means we did not properly handle postcompute stage somewhere.

Reply:
True yeah, fixed that.